Client-app/src/components/ClientProfile/index.js

```diff
@@ -32,9 +33,7 @@ function ClientProfile() {
   /**
    * Pull 'userInfoDetails' from the store (Redux centralized store)
    */
-  const userInfoDetails = useSelector(
-    (state) => state.userInfo.userInfoDetails
-  );
+  const userInfoDetails = useSelector(selectUserInfoDetails);
```

Comment:
An example of using "Selector" instead of writing the whole anonymous function like `(state) => state.storeName.stateName`